test/runTest.sh

```diff
@@ -117,6 +117,8 @@ function runPytests() {
 
   PYTHONPATH=${XPEDITE_DIR}:${PYTHONPATH} pytest ${COV} ${TEST_NAME} -v ${APP_HOST} ${TRANSACTION_COUNT} ${THREAD_COUNT} ${WORKSPACE} ${RUN_DIR_ARG} ${APPS}
 
+  ${TEST_DIR}/validateTarFiles.sh
```

Comment:
running test should not validate tar files. Why is this added ?

Reply:
I think these are issues from the merge, I will fix them.

Reply:
I put this here because the check is in generateBaseline.sh, but that script isn't run during the automated test. I can get rid of it.